crawler/stock_crawler.py

```diff
@@ -4,6 +4,13 @@
 from enum import Enum
 import datetime
 from django.core.serializers.json import DjangoJSONEncoder
+from multiprocessing import *
+
+from .multiprocessor_util import *
```

Comment:
Either need to refactor class to have independent startup method and instantiate class in `scheduled_task.py` script, change import to absolute path, or incorporate utility functions in same file.